dotnet-client-libraries/BenchmarkDotNetWithUnitTests/Tests/StringConcatBenchmarkLiveTest.cs

```diff
@@ -0,0 +1,53 @@
+namespace Tests;
+
+public class StringConcatBenchmarkLiveTest : IClassFixture<BenchmarkFixture>
+{
+    private readonly BenchmarkFixture _benchmarkFixture;
+
+    public StringConcatBenchmarkLiveTest(BenchmarkFixture benchmarkFixture)
+    {
+        _benchmarkFixture = benchmarkFixture;
+    }
+
+    [Fact]
+    public void Given_When_BenchmarkTests_Are_Run_Then_Six_Cases_Must_Be_Executed()
+    {
+        var benchmarkCases = _benchmarkFixture.BenchmarkSummary.Reports.Select(x => x.BenchmarkCase).ToList();
```

Comment:
This could be simplified to:
`var benchmarkCasesCount = _benchmarkFixture.BenchmarkSummary.Reports.Count(x => x.BenchmarkCase)`
and then the assertion could be changed to `Assert.Equal(6, benchmarkCasesCount)`

Reply:
If for consistency we wanted to keep this with the LINQ code then we can still simplify to this:
`var benchmarkCasesCount = _benchmarkFixture.BenchmarkSummary.Reports.Count(x => x.BenchmarkCase)`

we could even leave the lamba out and just call `Count()`, which under the hood is just going to then call the `Length` (at least I think - don't quote me as I haven't checked what all interfaces ImmutableArray implements).